filter_plugins/babylon.py

```diff
@@ -27,6 +29,25 @@ def filter_job_vars_secrets_to_dynamic_job_vars(anarchy_governor):
             dynamic_job_var_secrets.append(dynamic_job_var_secret)
     return dynamic_job_var_secrets
 
+def __insert_unvault_string(value, vaulted_values):
+    if isinstance(value, dict):
+        return { k: __insert_unvault_string(v, vaulted_values) for k, v in value.items() }
+    elif isinstance(value, list):
+        return [ __insert_unvault_string(v, vaulted_values) for v in value ]
+    if isinstance(value, str) and value.startswith('$ANSIBLE_VAULT;'):
+        letters = string.ascii_lowercase
+        vaulted_value_var = '__vaulted_value_' + ''.join(random.choice(string.ascii_lowercase + string.digits) for i in range(8))
```

Comment:
Why create a random name? Why not simply prefix the variable name with `__vaulted_value_`?